src/cdk-experimental/scrolling/auto-size-virtual-scroll.ts

```diff
@@ -151,6 +151,11 @@ export class AutoSizeVirtualScrollStrategy implements VirtualScrollStrategy {
     }
   }
 
+  /** Get the offset for the given index. */
+  getScrollOffsetForIndex() {
+    throw new Error('Offset location is not supported with autosize.');
```

Comment:
Let's say "not currently supported", this sounds like it never will be

Reply:
Does this mean that the scrollToIndex will not be supported for autosize in initial version?

Reply:
The autosize strategy will probably stay in cdk-experimental a little longer than the fixed size strategy, while we work out some of the bugs. This will be implemented before it goes into cdk